neqo-http3/src/features/extended_connect/webtransport_streams.rs

```diff
@@ -199,4 +199,28 @@ impl SendStream for WebTransportSendStream {
         }
         Ok(())
     }
+
+    fn stats(&mut self, conn: &mut Connection) -> Res<SendStreamStats> {
+        let stream_header_size = if self.stream_id.is_client_initiated() {
+            let id_len = if self.stream_id.is_uni() {
+                Encoder::varint_len(WEBTRANSPORT_UNI_STREAM)
+            } else {
+                Encoder::varint_len(WEBTRANSPORT_STREAM)
```

Comment:
These are constants, right?  I just checked and `Encoder::varint_len` can be made `const`.  That might push toward:

```rust
const TYPE_LEN_UNI: usize = Encoder::varint_len(WEBTRANSPORT_UNI_STREAM);
const TYPE_LEN_BIDI: usize = Encoder::varint_len(WEBTRANSPORT_STREAM);
let id_len = if self.stream_id.is_uni() { TYPE_LEN_UNI } else { TYPE_LEN_BIDI };
```